src/components/Autocomplete.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useMemo, useCallback } from 'react';
+import classNames from 'classnames';
+import debounce from 'lodash.debounce';
+import { Person } from '../types/Person';
+
+type Props = {
+  people: Person[];
+  selectedPerson?: Person | null;
+  onSelect?: (person: Person) => void;
+};
+
+export const AutoComplete: React.FC<Props> = ({
+  people,
+  selectedPerson,
+  onSelect = () => { },
+}) => {
+  const [visibleList, setVisibleList] = useState(false);
+  const [query, setQuery] = useState(selectedPerson?.name || '');
+  const [applieQuery, setApplieQuery] = useState('');
+
+  const applyQuery = useCallback(
+    debounce(setApplieQuery, 1000),
+    [],
```

Comment:
```suggestion
    [setAppliedQuery],
```
to fix it:
![image](https://github.com/mate-academy/react_autocomplete/assets/86923089/c716787b-4177-4f04-a688-b822e9d815e4)